.cirun.global.yml

```diff
@@ -1 +1,24 @@
-run_only_when_triggered_by: core
+# Self-Hosted Github Action Runners on Openstack via Cirun.io
+# Reference: https://docs.cirun.io/reference/yaml
+runners:
+  - name: cirun-openstack-gpu-large
+    # Cloud Provider: Openstack
+    cloud: openstack
+    # Instance type refers to flavors in openstack
+    instance_type: gpu_large
+    # The machine image to use for creating VM
+    machine_image: ubuntu-2204-nvidia-docker-20221229
```

Comment:
@aktech I'm going to merge here but I think this image name is not the latest one anymore, right? Can you confirm? Thanks!

Reply:
Correct, updated the image to latest: ubuntu-2204-nvidia-20230914